lib/install/create_hooks.dart

```diff
@@ -62,7 +62,7 @@ class CreateHooks {
       var hookTemplate = hookFile.readAsStringSync();
       var match =
           RegExp(r'dart\s(\S+)\s\$hookName').allMatches(hookTemplate).first;
-      commandPath = match.group(1);
+      commandPath = match.group(1)!;
```

Comment:
This will throw an error if it doesn't match. Should `commandPath` be an empty string if that's the case? Like:

```dart
commandPath = match.group(1) ?? '';
```